### PR TITLE
nixos/doc: document the firewall port ranges options

### LINKED ISSUE #32055: document how to allow port range in firewall
It works like this:

```
networking.firewall.allowedTCPPortRanges = [ { from = 4000; to = 4007; } ];
```
or `allowedUDPPortRanges` for UDP.

The information belongs here: https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/configuration/firewall.xml

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
Fix #32055

###### Things done

- [x] Built nixos-manual
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/doc/manual/configuration/firewall.xml [hunk 1]
```diff
@@ -25,8 +25,21 @@ networking.firewall.allowedTCPPorts = [ 80 443 ];
 Note that TCP port 22 (ssh) is opened automatically if the SSH daemon
 is enabled (<option>services.openssh.enable = true</option>).  UDP
 ports can be opened through
-<option>networking.firewall.allowedUDPPorts</option>.  Also of
-interest is
+<option>networking.firewall.allowedUDPPorts</option>.
+
+You can open ranges of TCP ports with:
```
Comment (FRidh) [inline nixos/doc/manual/configuration/firewall.xml]: Try not to use "You"

File: nixos/doc/manual/configuration/firewall.xml [hunk 2]
```diff
@@ -25,8 +25,21 @@ networking.firewall.allowedTCPPorts = [ 80 443 ];
...
+networking.firewall.allowedTCPPortRanges = [
+  { from = 4000; to = 4007; }
+  { from = 8000; to = 8010; }
+];
+</programlisting>
+
+Similarly, UDP port ranges can be opened through
+<option>networking.firewall.allowedUDPPortRanges</option>.
+
+Also of interest is
```
Comment (fpletz) [inline nixos/doc/manual/configuration/firewall.xml]: Note that this isn't markdown but docbook, so en emtpy line will not produce a new paragraph. You need to close the para tag and open a new one.

Comment (rnhmjoj) [reply nixos/doc/manual/configuration/firewall.xml]: Better?

**Comment by rnhmjoj** [comment]
Update?

**Comment by orivej** [comment]
Thank you!

**Pull request merged**